invoker.js

```diff
@@ -313,7 +313,7 @@
       const shouldHide = !canShow && command === "hide-popover";
 
       if (shouldShow) {
-        invokee.showPopover();
+        invokee.showPopover({ source: source });
```

Comment:
This is cool, but do you know what's cooler?

```suggestion
        invokee.showPopover({ source });
```